bundles/org.openhab.binding.proteusecometer/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab-addons
```

Comment:
Have a look at other NOTICE files as you will need to add some lines for jSerialComm library.

example:

== Third-party Content

oshi-core
* License: MIT License
* Project: https://github.com/oshi/oshi
* Source:  https://github.com/oshi/oshi